handbook/communication/seeking-and-giving-feedback.md

```diff
@@ -0,0 +1,50 @@
+# Seeking and giving feedback
+
+Seeking and acting on feedback is crucial for making good decisions and delivering our best work. Good feedback is a product of an intentional environment with positive feedback loops.
+
+## Seeking feedback
+
+When seeking feedback at Sourcegraph, it's important to:
+
+- Specifically say what *kind* of feedback you're seeking. A thumbs-up, thumbs-down? A gut call on the better of two approaches? A detailed critique?
+- Ask *specific questions* that you want feedback about.
+- Provide *context*. Make sure the specific problem and goal is clear.
+- Seek feedback according to [project roles](#project-roles).
+- After receiving feedback, follow up. Recognize contributions and share how you will move forward.
+
+### Great examples of requesting feedback
+
+(To be added!)
+
+## Giving feedback
+
+When giving feedback, it's important to:
+
+- Make sure you understand what kind of feedback is sought. If it's not clear what kind of feedback is sought, clarify before giving feedback.
+- Focus feedback on the specific feedback sought. Other feedback may be valuable, but should be separated into a distinct conversation.
+- Provide feedback in terms of **why**, not **how** (unless specifically sought).
+- Clarify where the feedback is coming from: is it based on a heuristic or best practice? Is it from a previous decision? Is it rooted in personal opinion?
+- Focus on the work, not the individual.
+
+Keep in mind that providing ongoing feedback is distinct from [disagreeing with a decision](code_of_conduct.md#conflict-resolution). Typically, feedback is sought when a decision is yet to be made.
+
+### Great examples of providing feedback
+
+(To be added!)
+
+## Project roles
+
+When a project has multiple stakeholders, it's useful to identify specific levels of participation through project roles. Each project should clearly state who is:
+
+1. **Responsible:** Those who do the work to get things done.
```

Comment:
Should we add those categories of stakeholders to the RFC templated @quinnkeast @christinaforney ?

Reply:
I _think_ we talked about this when RACI roles first came up, and we agreed that if people find them useful, they'll start being used. Personally, I think they're only useful when an RFC reaches a certain level of definition (e.g. defined project and team). Too early, and they get in the way of defining the problem and proposing a solution.